pkg/secrets/info_windows.go

```diff
@@ -10,41 +10,46 @@ package secrets
 
 import (
 	"bytes"
+	_ "embed"
 	"fmt"
 	"os/exec"
 	"strings"
 )
 
-func (info *SecretInfo) populateRights() {
-	execPath := fmt.Sprintf("\"%s\"", strings.TrimSpace(info.ExecutablePath))
-	err := checkRights(execPath, secretBackendCommandAllowGroupExec)
-	if err != nil {
-		info.Rights = fmt.Sprintf("Error: %s", err)
-	} else {
-		info.Rights = fmt.Sprintf("OK, the executable has the correct rights")
-	}
+//go:embed info_win.tmpl
+var rightsDetailsTemplate string
+
+type rightDetails struct {
+	Error  string
+	Stdout string
+	Stderr string
+}
 
+func getExecutableRights() (interface{}, error) {
+	execPath := fmt.Sprintf("\"%s\"", strings.TrimSpace(secretBackendCommand))
 	ps, err := exec.LookPath("powershell.exe")
 	if err != nil {
-		info.RightDetails = fmt.Sprintf("Could not find executable powershell.exe: %s", err)
-		return
+		return nil, fmt.Errorf("Could not find executable powershell.exe: %s", err)
 	}
 
-	cmd := exec.Command(ps, "get-acl", "-Path", execPath, "|", "format-list")
-
 	stdout := bytes.Buffer{}
 	stderr := bytes.Buffer{}
+	details := rightDetails{}
```

Comment:
`right` has many meanings (direction, correctness, identification) and in this case also the permissions so I think that the name as you use it in different parts of the PR needs an alternative (maybe permission/access/etc?).